<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchTable</class>
 <widget class="QDialog" name="MatchTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>384</height>
   </rect>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QPushButton" name="bNewMatch">
        <property name="text">
         <string>New Match</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/add_match.png</normaloff>:/images/add_match.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bEditMatch">
        <property name="text">
         <string>Edit Match</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/edit_match.png</normaloff>:/images/edit_match.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bMatchList">
        <property name="text">
         <string>Match List</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/matches.png</normaloff>:/images/matches.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cAgeGroup">
        <property name="text">
         <string>Age Group</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tMatches"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
